### PR TITLE
Chapter 5 revision

### LINKED ISSUE #351: Data licensing agreements AND include primary data from partners
We need to cover this -- better to do so in Ch1, Ch5 or Ch7? 
---

Add here the importance of establishing a data licensing agreement if directly acquiring data from partners (e.g. administrative data that is not otherwise public, or proprietary data from firms, etc).

_Originally posted by @mariaruth in https://github.com/worldbank/d4di/pull/293_

### PR DESCRIPTION
**Pull request by bbdaniels**
This is not 100% done but I am happy here. Ready for review. @mariaruth should do the introduction which I have not touched. @kbjarkefur should review the data security portions and add throughout as needed.

For content, @mariaruth should add links to resources for licensing and ownership agreements as appropriate. I have not done this, but it is ready for review.

Resolves #351.

### CODE REVIEW

**Comment by luizaandrade** [comment]
I'm consolidating all the changes to data security content into one PR. Merged this to the new `data-security` branch in #367

**Comment by luizaandrade** [comment]
This PR includes changes that were not discussed last time we met and that conflict with the assignments that Kris and I had. We now cannot merge changes related to #327 and #343 without creating quite complex conflicts. So we either leave these changes for later, or we will not be able to have a version for Arianna to review this week.

**Comment by mariaruth** [comment]
@luizaandrade not sure i understand the comment above. should i wait to review / add to this until the conflicts are resolved?

**Comment by kbjarkefur** [comment]
I added my encryption edits from chapter 1 even though it is a ch 5 branch. Just trying to reduce the number of open branches by consolidating them

**Comment by bbdaniels** [comment]
> This PR includes changes that were not discussed last time we met and that conflict with the assignments that Kris and I had.
> 
> PRs with changes related to #327 and #343 will create complex conflicts. If we want to have a version ready for review on Monday, we may have to choose what to include.

Agree — we can discuss best resolution. I didn't realize other people were working on Ch5 in ways that this would conflict so much. One solution for now is to just take the new section in and hold off on the reorganizing, but let's look at it Tuesday and take a call together. This should be the last thing open?

File: chapters/data-collection.tex [hunk 1]
```diff
@@ -13,109 +13,337 @@
 \end{fullwidth}
 
 %------------------------------------------------
-\section{Survey development workflow}
-A well-designed questionnaire results from careful planning, consideration of analysis and indicators, close review of existing questionnaires, survey pilots, and research team and stakeholder review. There are many excellent resources on questionnaire design, such as from the World Bank's Living Standards Measurement Survey.
-\sidenote{Grosh, Margaret; Glewwe, Paul. 2000. Designing Household Survey Questionnaires for Developing Countries : Lessons from 15 Years of the Living Standards Measurement Study, Volume 3. Washington, DC: World Bank. © World Bank.\url{https://openknowledge.worldbank.org/handle/10986/15195 License: CC BY 3.0 IGO.}}
-The focus of this chapter is the particular design challenges for electronic surveys (often referred to as Computer Assisted Personal Interviews (CAPI)).
-\sidenote{\url{https://dimewiki.worldbank.org/wiki/Computer-Assisted\_Personal\_Interviews\_(CAPI)}}
+\section{Collecting primary data with development partners}
```
Comment (kbjarkefur) [inline chapters/data-collection.tex]: Can we meet and agree on what we want to say in this section? This is the only section of this chapter (maybe the whole book) with no sources or sidenotes, and it feels like it was started to be written before we had an idea of what our recommendation. I think we should leave this as it is now, resolve all the conflicts this re-organization causes, and then meet and decide what we are recommending, and then and only then worry about the text. How does that sound?

**Comment by kbjarkefur** [comment]
Anytime anyone wants to re-organize a full chapter, I think it is that persons responsibility to make sure that is communicated to everyone else in the team that are equally busy working towards their deadlines.

**Comment by bbdaniels** [comment]
Agree

https://github.com/worldbank/d4di/issues/351#issuecomment-583475942

**Comment by kbjarkefur** [comment]
Use slack for that next time, or at least tag people it can have a consequence for. Neither Luiza or I were subscribed to that issue.

**Comment by kbjarkefur** [comment]
Ok, I think all my edits are in, and I included @bbdaniels  's edits that I liked. Hopefully I have not forgotten too much stuff. This is ready enough for @luizaandrade to take over

**Pull request merged**